RELEASING.adoc

```diff
@@ -1,80 +1,177 @@
+ # Identifies the current build.
+ # These will be embedded in the app and displayed when it starts.
+-VERSION ?= v0.5.1-SNAPSHOT
++VERSION ?= v1.0.0
+ COMMIT_HASH ?= $(shell git rev-parse HEAD)
 
-For example,
+ # Indicates which version of the UI console is to be embedded
+----
 
-```
-VERSION ?= v1.0.0
-...
-CONSOLE_VERSION ?= 0.2.11
-```
+By default, the Makefile gets the latest version of the UI, so we'll also
+change the console version on the Makefile:
 
-2) Commit that change to your private GitHub repository:
+[source, diff]
+----
+diff --git a/Makefile b/Makefile
+index c1f7f80..1d05ff3 100644
+--- a/Makefile
++++ b/Makefile
+@@ -13,7 +13,7 @@ COMMIT_HASH ?= $(shell git rev-parse HEAD)
+ # WARNING: If you have previously run the 'docker' target but
+ # later want to change the CONSOLE_VERSION then you must run
+ # the 'clean' target first before re-running the 'docker' target.
+-CONSOLE_VERSION ?= latest
++CONSOLE_VERSION ?= v1.0.0
+ CONSOLE_LOCAL_DIR ?= ../../../../../kiali-ui
 
-```
-$ git add Makefile
-$ git commit -m "New release"
-$ git push <your remote repo name> master
-```
+ # Version label is used in the OpenShift/K8S resources to identify
+----
 
-3) Create a GitHub PR, have someone review it, and then merge it into Kiali `master` branch.
+== Creating the release PR
 
-At this point, Travis will run the test suite, build the binary, and push the "latest" docker image to link:https://hub.docker.com/r/kiali/kiali[DockerHub].
+[source, bash]
+----
+$ git add Makefile swagger.json
+$ git commit -m "v1.0.0 Release"
+$ git push <your remote> master
+----
 
-4) Once Travis is "green" (which confirms all tests have passed and the latest image has been pushed to DockerHub), you can now pull the master branch (to pull in the PR merge commit) and tag the release.
+Create a GitHub PR, have someone review it, and then merge it.
 
-NOTE: You must have write permissions to the public GitHub repository in order to be able to do this.
+After the merge, Travis will run the test suite, build the binary, and push the
+"latest" docker image to link:https://hub.docker.com/r/kiali/kiali[DockerHub].
+
+== Tag Git Repo
 
-For example,
+Now you need to pull the latest master with the merged commit, and create the
+tag for it.
 
-```
+[source, bash]
+----
 $ git pull
-$ git tag v1.0.0
+$ git tag v1.0.0 <merge commit id hash>
 $ git push origin v1.0.0
-```
+----
+
+Once the tag has been pushed, Travis will immediately run another build, but
+this time will tag the docker image with the same version string as the git tag
+name. In the example above, the docker image pushed to DockerHub will not be
+tagged with "latest" but will instead be tagged with "v1.0.0".
+
+You can let the committers know that the release has been completed and PRs can
+begin to be merged again. Though, it would be best to wait for this
+notification to go out until after you bump up the version string to the next
+snapshot version (see "Prepare For The Next Release" below).
```

Comment:
Looks like this "Prepare For The Next Release" section has been deleted in this PR. I think we need to add that back - at the end of the release process, we have to tell the reader that they need to set the Makefile back to CONSOLE_VERSION=latest and VERSION=<the next version-SNAPSHOT> to get ready for the next iteration of development.

Reply:
Done, added it back.